tests/DS/test_sds_eval.sh

```diff
@@ -162,7 +162,7 @@ test_run "sds_external_xccdf" test_sds_external_xccdf sds_external_xccdf sds_ext
 test_run "sds_tailoring" test_sds_tailoring sds_tailoring sds_tailoring/sds.ds.xml scap_com.example_datastream_with_tailoring xccdf_com.example_cref_tailoring_01 xccdf_com.example_profile_tailoring
 
 test_run "eval_simple" test_eval eval_simple/sds.xml
-test_run "cpe_in_ds" test_eval cpe_in_ds/sds.xml
+test -f "/etc/sysconfig/prelink" && test_run "cpe_in_ds" test_eval cpe_in_ds/sds.xml
```

Comment:
`return 255`, like in previous case.

Reply:
I'm concerned that there are 20 different tests in this file and a "return" will skip all of them when only this one test is incompatible.  Since this test isn't a function, the return would be at the top level.  I feel a better solution would be to modify the cpe_in_ds/sds.xml file to account for the location of the prelink configuration if installed in Debian (/etc/default/prelink) which uses the same configuration variable (PRELINKING=no).  I'm not familiar enough with how to expand that xml file.

Reply:
Yeah, that. Or you could just wrap it.

Reply:
@ggbecker or @jan-cerny could be able to help with adopting it to Debian's `/etc/default/prelink`, I think.

Reply:
Can you please provide the output you got from this test? I'm trying to figure out what could be wrong it and I'm kinda lost. It might be that the test is actually broken.

Reply:
```
113: + cpe_in_ds                                                   
113: TEST: cpe_in_ds
113: TitPrelinking Disabled
113: Rulxccdf_org.ssgproject.content_rule_disable_prelink
113: Resnotapplicable
113: 
113: 0a1
113: > W: oscap:     OVAL object not supported.
1/1 Test #113: DS/test_ds_misc.sh ...............***Failed   17.54 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =  17.55 sec

The following tests FAILED:
	113 - DS/test_ds_misc.sh (Failed)
Errors while running CTest
```

Reply:
Uploads won't work but here's the "DEVEL" level output: https://pastebin.com/DUtxd1HH

Reply:
This may be the actual root cause of the test failing:
```
113: > I: oscap:     Entity 'family'='unix' of item '11259603' matches corresponding entity in state 'oval:ssg:ste:111'. [oscap(125960):oscap(7fe38cd35b80):oval_resultTest.c:636:eval_item]
113: > I: oscap:     Item '11259603' compared to state 'oval:ssg:ste:111' with result true. [oscap(125960):oscap(7fe38cd35b80):oval_resultTest.c:668:eval_item]
113: > I: oscap:   Test 'oval:ssg:tst:101' evaluated as true. [oscap(125960):oscap(7fe38cd35b80):oval_resultTest.c:1148:oval_result_test_eval]
113: > I: oscap:   Evaluating rpminfo test 'oval:ssg:tst:102': fedora-release is version 1. [oscap(125960):oscap(7fe38cd35b80):oval_resultTest.c:1129:oval_result_test_eval]
```

It appears to be evaluating as Fedora 1

Reply:
It looks like the entire `cpe_in_ds` uses a Fedora specific SSG benchmark.  I'm not sure if it's appropriate to adjust that benchmark to support other distributions.  I assume a different benchmark would be needed or that subtest be conditional.  It currently is conditional in this PR but based on a target file, not based on the platform.

Reply:
OK. So here is the story. This test is about checking the DataStream features, particularly the CPE one. The DS in the test is actually only applicable for Fedora 1 or RHEL6 / RHEL7, so it is `not applicable` on everything contemporary, regardless of existence of the file, because of this CPE check. But it's all right since the point of the test is to check that CPE features of the DS are not crating any errors in the scanner (hence is the check for `stderr` to be empty in the test).

The problem is: the DS in the test uses RPM-specific check for OS version and it is generating the error. We should guard this test with `probecheck "rpminfo" || return 255` instead of the file presence check. It is *for now* solution. And then, at some point the future, we should re-write it to be distribution-neutral.